cg/services/order_validation_service/rules/sample/rules.py

```diff
@@ -77,7 +71,7 @@ def validate_required_volume(
 
 
 def validate_organism_exists(
-    order: OrderWithNonHumanSamples, store: Store, **kwargs
+    order: OrderWithSamples, store: Store, **kwargs
```

Comment:
Are we planning to address the introduced warnings in a separate PR? I can do it after you merge.

Reply:
do it in a separate PR, not sure how to solve them